Jenkinsfile

```diff
@@ -315,6 +315,16 @@ try {
                 Utils.markStageSkippedForConditional("tpcdsQueryPlansAndVerification")
               }
             }
+          }, jobQueryPlans: {
```

Comment:
Probably to get better more accurate plots with plot_operator_breakdown.py ? Would this make sense for TPC-DS or JOB as well?

Reply:
Good question, I have no idea. Sometimes, we also get different plans for the second run for TPC-H because different predicates can lead to different cardinalities and thereby reorder operators. 

I don't know whether two runs that also visualize the plans would make the operator breakdowns much more accurate.

Reply:
Do you have any experience with that?

Reply:
> I don't know whether two runs that also visualize the plans would make the operator breakdowns much more accurate.

The second run should override the SVGs, so plot_operator_breakdown.py will plot the numbers of the second run.

> Do you have any experience with that?

No, I was just guessing.

Reply:
> The second run should override the SVGs, so plot_operator_breakdown.py will plot the numbers of the second run.

Yes, the runtime are likely to be much shorter due to caching etc. However, I don't know if it's worth the additional build time because the CI's operator breakdowns are not too reliable anyways because of parallel builds and so on.

> No, I was just guessing.

I will test it quickly.

Reply:
The TPC-DS run is actually used to create and plans **and** verify the results, I would not add a second run here. Interestingly the runtimes for the performance breakdown hardly changes (sorry for the color mismatch). For these reasons I would stick with the current version.

[operator_breakdown_two_runs.pdf](https://github.com/hyrise/hyrise/files/6887226/operator_breakdown_two_runs.pdf)
[operator_breakdown.pdf](https://github.com/hyrise/hyrise/files/6887227/operator_breakdown.pdf)

Reply:
Great, thanks for checking!

Reply:
You did not yet change it to `-r 1`. Are you still testing?

Reply:
I tested the effect on TPC-DS. For TPC-H I would keep it as it is because it does not take much extra time and I assume that it was initially set to `-r 2` on purpose (even though we don't exactly know what the reason was).